### PR TITLE
[Snyk] Fix for 6 vulnerable dependencies

### PR DESCRIPTION
**Pull request by PatOnTheBack**
Upgraded packages to fix vulnerabilities.

Before submitting your PR please make sure:

* [X] You worked on the develop branch (or any other branch which was forked from develop),
* [X] Your PR is not targeting the master branch,
* [ ] You tested your code and it works.

Thank you!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,18 +33,18 @@
   "dependencies": {
     "@shagstrom/split-pane": "0.9.4",
     "celldown.js": "0.1.0",
-    "chokidar": "1.6.1",
+    "chokidar": "2.0.0",
```
Comment (brrd) [inline package.json]: New major versions generally introduce breaking changes. Therefore we can't move to chokidar 2 unless we make sure this is actually working (and fix things if this doesn't). This PR need further investigation IMO.

Comment (PatOnTheBack) [reply package.json]: I agree. For now, should you just merge the PR with all the other fixes?

Comment (brrd) [reply package.json]: :+1: Feel free to update your PR (and please be careful of merge conflicts).

**Comment by PatOnTheBack** [comment]
Sorry, I don't know why `jquery` lost its new version number in `package.json`. I fixed it. `is-url` was at the correct version number.

**Comment by brrd** [comment]
Thanks

**Pull request merged**